### PR TITLE
Update readme 

### PR DESCRIPTION
**Pull request by jenniferwills**
- Update README to reflect some important things during setup + make it more obvious we want ```npm run format``` run.
- Added an example env file .env_example to show the necessary format and make it easier to get up and running with the Google Calendar APIs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,14 @@ Getting started on your own!!
...
+5. Remember to run [npm run format](#npm-run-format) before you a PR up for review
+
+6. There are also some environment files you must create (and should never be committed to git):
+  - Google Calendar API:
+    - make a copy of the .env_example file in the root directory public-website and name the copy .env (this is in the .gitignore and should not be committed to git)
+    - get the Calendar ID from google calendar and set the variable REACT_APP_GOOGLE_CALENDAR_ID to it (if you don't have access to the [SE Events](https://calendar.google.com/calendar?cid=bGI2dHRiMWtzMzdxbDVyOGlzdWFzZ2NkbzhAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) calendar contact @RobbieZhuang, @jenniferwills or @yingninggui, and we can help) 
+      - to get the Calendar ID if you have access to the SE Soc Events calendar go to settings and sharing (for the calendar) -> scroll down to “Integrate Calendar” section -> Calendar ID (this the ID you want)
+    - Get a [Google API Key](https://developers.google.com/calendar/quickstart/js) (make sure to follow the entirety of step 1, you may need to enable google calendar API before generating a key) from google calendar and set REACT_APP_GOOGLE_CALENDAR_API_KEY to it.
```
Comment (RobbieZhuang) [inline README.md:13]: mention: click the **Create API Key** button

File: README.md [hunk 2]
```diff
@@ -3,6 +3,14 @@ Getting started on your own!!
 2. cd public-website
 3. npm install
 4. npm start (this creates a new local server running the website, and will open a new browser window)
+5. Remember to run [npm run format](#npm-run-format) before you a PR up for review
+
+6. There are also some environment files you must create (and should never be committed to git):
+  - Google Calendar API:
+    - make a copy of the .env_example file in the root directory public-website and name the copy .env (this is in the .gitignore and should not be committed to git)
+    - get the Calendar ID from google calendar and set the variable REACT_APP_GOOGLE_CALENDAR_ID to it (if you don't have access to the [SE Events](https://calendar.google.com/calendar?cid=bGI2dHRiMWtzMzdxbDVyOGlzdWFzZ2NkbzhAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) calendar contact @RobbieZhuang, @jenniferwills or @yingninggui, and we can help) 
+      - to get the Calendar ID if you have access to the SE Soc Events calendar go to settings and sharing (for the calendar) -> scroll down to “Integrate Calendar” section -> Calendar ID (this the ID you want)
```
Comment (RobbieZhuang) [inline README.md:12]: nit: remove `if you have access to the SE Soc Events calendar`

File: README.md [hunk 3]
```diff
@@ -3,6 +3,14 @@ Getting started on your own!!
 2. cd public-website
 3. npm install
 4. npm start (this creates a new local server running the website, and will open a new browser window)
+5. Remember to run [npm run format](#npm-run-format) before you a PR up for review
+
+6. There are also some environment files you must create (and should never be committed to git):
+  - Google Calendar API:
+    - make a copy of the .env_example file in the root directory public-website and name the copy .env (this is in the .gitignore and should not be committed to git)
+    - get the Calendar ID from google calendar and set the variable REACT_APP_GOOGLE_CALENDAR_ID to it (if you don't have access to the [SE Events](https://calendar.google.com/calendar?cid=bGI2dHRiMWtzMzdxbDVyOGlzdWFzZ2NkbzhAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) calendar contact @RobbieZhuang, @jenniferwills or @yingninggui, and we can help)
```
Comment (RobbieZhuang) [inline README.md:11]: nit: You could also add in a line about using one's own test calendars for dev work

Comment (jenniferwills) [reply README.md:11]: That would simply be replace the Calendar ID to their own calendar right?

**Pull request merged**